app/support/reservations/validations.rb

```diff
@@ -21,10 +21,22 @@ module Reservations::Validations
 
     validate :starts_before_ends
     validate :duration_is_interval
+    validate :category_is_valid
   end
 
   # Validation Methods
 
+  def category_is_valid
+    return if category.blank?
+    if self.class.const_defined?(:CATEGORIES)
```

Comment:
And should we be allowing blank category on offline? I think it's a fine assumption for now, but I could see that changing.